changelog.md

```diff
@@ -9,6 +9,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 - Send bug reports and app crashes details to Sentry
 - Clickable checkbox rows
+- Fetch restaurants from sigarra and stored locally using action creator
```

Comment:
Can you reformulate this to  `Fetch restaurants' cantines from sigarra and storage?` it should be agnostic of technologies